src/components/MovieTile.jsx

```diff
@@ -1,24 +1,67 @@
+import { useRef, useState, useEffect } from "react";
 import PropTypes from 'prop-types';
 import { getReleaseYear } from '../utils/movieUtils';
-import '../components-styles/MovieTile.scss';
+import '../styles/MovieTile.scss';
+import DialogOpenButton from '../components/DialogOpenButton';
 
-const MovieTile = ({poster_path, title, genres, release_date, onClick}) => {
-    const handleClickEvent = (e) => {
-        if (typeof onClick === "function") {
-            onClick(e.currentTarget.value)
+const useOutsideClick = (menuRef, setMenuTileContextOpen) => {
```

Comment:
Lets create this state value inside component and return it value as boolean

Reply:
@MykhailoSheliahov there is also an event on button "close" that changes the state